src/css/pages.css

```
@@ -38122,7 +38122,7 @@ tr.show-desktop {
     transition: none !important;
   }
 }
-._1ju7xj0j {
+.btn-outline--small {
```

```suggestion
.btn-outline-small {
```
di nyo po naupdate lahat

---

I see. Thanks for this. This is due to merge conflict earlier.